app/router.js

```diff
@@ -14,7 +14,7 @@ Router.map(function () {
     this.route('cookiestatement');
     this.route('accessibilitystatement');
   });
-  this.route('regulatory-attachments', function () {
+  this.route('templates-management', function () {
```

Comment:
I feel bad for saying this, but my brain can't let it go with all the related file names as well... I think it should be 'template-management' to be correct English...

Reply:
Yes agreed, we should then also rename `codelists-management` to `codelist-management` and `snippets-management` to `snippet-management`. I'll do the renames in a seperate PR.

Reply:
Should be adjusted in https://github.com/lblod/frontend-reglementaire-bijlage/pull/238 :)